d2l-table-style.js

```diff
@@ -51,12 +39,9 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-table-style">
 				text-align: left;
 				vertical-align: middle;
 			}
-
-			d2l-table-wrapper[type="light"] .d2l-table > * > tr > td,
-			d2l-table-wrapper[type="light"] .d2l-table > * > tr > th,
 			d2l-table[type="light"] d2l-td,
 			d2l-table[type="light"] d2l-th {
-				border-top: var(--d2l-table-light-border);
+				border-top: var(--d2l-table-border);
```

Comment:
Cleaned up some CSS variable use while I was in here to better mirror the variables used in the Lit versions.